### DISCUSSION TITLE
What does shards_num mean? Is it applicable to milvus standalone?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#18389] [Q&A and General discussion]
What does shards_num mean? Is it applicable to milvus standalone?
What number should I choose?

**Comment by martinenkoEduard** [comment]
Should I use 1 or more?

**Comment by yhmo** [comment]
For each insert action, milvus will split the inserted data into several pieces according to shard_num. Each piece data will be put into a segment. So, if shard_num=2, then for each insert action, the inserted data will be consumed by two segments.
The default value of shard_num is 2.
If there are multiple data nodes in a cluster, milvus will assign shard to different node. For example, you deploy 4 data nodes in a milvus cluster, then you can set the shard_num=4, each data node will process the data parallally.
Standalone no need to change the shard_nu, just keep the default value.
